codex-ui/dev/index.html

```diff
@@ -2,6 +2,7 @@
   <title>Codex UI</title>
 </head>
 <body
+  mode="dark"
```

Comment:
I'm afraid that `mode` could be used by user's application for some other stuff, let's make our property more specific, like `color-scheme`